database.yml

```diff
@@ -0,0 +1,31 @@
+#  Copyright (c) 2012-2013, Jungwacht Blauring Schweiz. This file is part of
+#  hitobito and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito.
+
+base: &generic
+  adapter: <%= ENV['RAILS_DB_ADAPTER'] || 'sqlite3' %>
+  pool: 20
+  timeout: 5000
+  encoding: utf8
+<% %w(host port username password socket).each do |option| %>
+  <% value = ENV["RAILS_DB_#{option.upcase}"] %>
+  <%= "#{option}: #{value}" if value.present? %>
+<% end %>
+
+
+development:
+  <<: *generic
+  pool: 1
+  database: <%= ENV['RAILS_DB_NAME'] || "#{Rails.root}/db/development.sqlite3" %>
+
+# Warning: The database defined as "test" will be erased and
+# re-generated from your development database when you run "rake".
+# Do not set this db to the same as development or production.
+test:
+  <<: *generic
+  database: <%= ENV['RAILS_TEST_DB_NAME'] || ENV['RAILS_DB_NAME'] || "#{Rails.root}/db/test.sqlite3" %>
```

Comment:
Ich nehme an, auch wenn wir beim command `-e test` beim launch drin haben nimmt er development (Siehe [hier]( https://github.com/cleverer/hitobito-docker/blob/cypress/docker-compose.yml#L58))? Oder ist das ein Versehen?

Reply:
Ich habe einfach in der Rails-Konsole herausgefunden dass aktuell in cypressserver `Rails.env === 'development'` ist. Habe auch gerade nicht den Überblick wo das überall gesetzt und überschrieben wird.

Reply:
Ah ok. ich hatte gemeint das `-e test` geht auch, aber anscheinend kann man das nur mit `RAILS_ENV` setzen. Das würde aber denn seed überspringen. Ist ja ein POC, kann man so lassen.